server.js

```diff
@@ -11,13 +14,38 @@ const welcomeMessage = {
   text: "Welcome to CYF chat system!",
 };
 
+const PORT = 9002;
+
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
+
+app
+  .get("/", function (request, response) {
+    response.sendFile(__dirname + "/index.html");
+  })
+  .post("/messages", function (request, response) {
```

Comment:
when using routes with parameters, they should be ordered from the most specific to the least specific (see article here https://gabrieleromanato.name/order-of-routes-when-using-parameters-in-expressjs), so `/messages` routes should come after `/messages/:id` routes